src/components/graph-controls.js

```diff
@@ -62,10 +73,36 @@ function GraphControls({
     [minZoom, maxZoom, zoomLevel, modifyZoom]
   );
 
+  const handleDocumentClick = e => {
+    if (!helpContainerRef.current.contains(e.target)) {
+      setHelpShowing(false);
+    }
+  };
+
+  useEffect(() => {
+    if (!showHelp) {
+      document.removeEventListener('click', handleDocumentClick);
```

Comment:
Curious, can this be accomplished with mouse events on help div (ex. onMouseLeave)?

Reply:
I don't want the help menu to close when the mouse leaves, only when the document, or anything not related to the help container, is clicked. Otherwise the help menu would close at the slightest mouse movement.